spec/shopify_transporter/pipeline/magento/product/top_level_variant_attributes_spec.rb

```diff
@@ -9,14 +9,13 @@ module ShopifyTransporter::Pipeline::Magento::Product
         child_product = FactoryBot.build(:advanced_simple_product)
         parent_product = FactoryBot.build(:advanced_configurable_product, {variants: [child_product]})
         variants_in_shopify_format = described_class.new.convert(child_product, parent_product)
-        expected_variants_in_shopify_format = [
-            {
-              sku: 'm100',
-              grams: '100',
-              price: '222'
-            }.deep_stringify_keys
-          ]
-        expect(expected_variants_in_shopify_format).to eq(variants_in_shopify_format)
+        expected_variants_in_shopify_format = {
+          sku: child_product['sku'],
+          grams: child_product['weight'],
+          price: child_product['price']
+        }.deep_stringify_keys
+
+        expect(variants_in_shopify_format).to include(expected_variants_in_shopify_format)
```

Comment:
Would prefer the stringify keys to be on this line vs above as we usually do in other tests (it's been generally helpful too, IMO).